README.md

```diff
@@ -77,4 +77,4 @@ may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
 Unless required by applicable law or agreed to in writing, software distributed
 under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
 CONDITIONS OF ANY KIND, either express or implied. See the License for the
-specific language governing permissions and limitations under the License.
```

Comment:
fix- not maintainable

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Not maintainable

Reply:
Looks good and maintainable

---
**Sema Reaction:** :ok_hand: This code looks good | **Sema Tags:** Maintainable

Reply:
tech debt

---
**Sema Tags:** Readable, Not secure, Elegant, Brittle, Maintainable